tests/v2/actions/etcdsnapshot/verify.go

```diff
@@ -16,25 +15,59 @@ import (
 	kwait "k8s.io/apimachinery/pkg/util/wait"
 )
 
-// VerifySnapshots verifies that all snapshots come to an active state, and the correct number
+const (
+	expectedRKE1SnapshotLen = 1
+)
+
+// VerifyRKE1Snapshots verifies that all snapshots come to an active state, and the correct number
+// of snapshots were taken based on the number of nodes and snapshot type (s3 vs. local)
+func VerifyRKE1Snapshots(client *rancher.Client, clusterName string, snapshotIDs []string) error {
+	err := kwait.PollUntilContextTimeout(context.TODO(), 5*time.Second, extdefault.FiveMinuteTimeout, true, func(ctx context.Context) (done bool, err error) {
+		var snapshotObj any
+		var state string
+
+		for _, snapshotID := range snapshotIDs {
+			snapshotObj, err = client.Management.EtcdBackup.ByID(snapshotID)
+			state = snapshotObj.(*management.EtcdBackup).State
+
+			if err != nil {
+				return false, nil
+			}
+
+			if state != "active" {
+				return false, nil
+			}
+		}
+		return true, nil
+	})
+	if err != nil {
+		return err
+	}
+
+	// RKE1 doesn't give us snapshot-per-node. It gives us one object representing the snapshot for the cluster.
```

Comment:
if the cluster already had snapshots taken and in the test we retake a new snapshot the length would still be 1?